assets/src/edit-story/components/header/buttons/update.js

```diff
@@ -72,21 +80,31 @@ function Update() {
     default:
       text = __('Save draft', 'web-stories');
       return (
-        <Outline
-          onClick={() => saveStory({ status: 'draft' })}
-          isDisabled={
-            !hasMetaBoxes && (isSaving || isUploading || !hasNewChanges)
-          }
+        <Tooltip
+          title={text}
+          placement={TOOLTIP_PLACEMENT.BOTTOM}
+          hasTail={true}
         >
-          {text}
-        </Outline>
+          <Button
+            variant={BUTTON_VARIANTS.SQUARE}
+            type={BUTTON_TYPES.TERTIARY}
+            size={BUTTON_SIZES.SMALL}
+            onClick={() => saveStory({ status: 'draft' })}
+            disabled={
+              !hasMetaBoxes && (isSaving || isUploading || !hasNewChanges)
```

Comment:
This one has to be updated. If the user is saving or uploading, the button _**has**_ to be disabled. You shouldn't be able to save under those circumstances regardless of the presence of meta-boxes. It's only `hasNewChanges` that can be overridden by `hasMetaboxes`:

```js
              isSaving || isUploading || (!hasNewChanges && !hasMetaBoxes)
```

But it might make sense to move this to a (positively named) variable, pull it up top, and affix a wordy comment about what's in here and why. I hate getting confused by these - especially negative flags 😂 😂 .

So, something like:

```js
  // The button is enabled only if we're not already saving nor uploading. And
  // then only if there are new changes or the story has meta-boxes – as they
  // can update without us knowing it.
  const isEnabled = !isSaving && !isUploading && (hasNewChanges || hasMetaBoxes)
  ...
  disabled={!isEnabled}
```